### ISSUE TITLE
Two finger tap doesn't right click.

### ISSUE THREAD

**Issue by michaelstephens** [SicVolo/hid-apple-3.19#4]
This might be a case of PEBCAK but actually clicking (can it even be called clicking on a force touch trackpad?) does the desired right click, but just tapping with two fingers does not right click. Tapping with one finger does indeed single click just fine. 

I am wondering if this feature is working and the error is on my end or if it is not yet implemented.

PS. Thank you so much for your work. This is a must have for the 2015 MBPs.

**Comment by SicVolo** [comment]
The double finger tap will work once these module are compiled and installed per the README.
Also the changes have been merged into the kernel 4.2-rc5, so no modules will be needed if you get that kernel or later

**Comment by michaelstephens** [comment]
Yes, I followed the steps in the DKMS Installation method. Should I have manually done it?

**Comment by SicVolo** [comment]
Yes

**Comment by michaelstephens** [comment]
Thought I'd leave you an update. I uninstalled my previous installation via DKMS and followed the manual installation steps. Still no double finger tap.

Either way its a minor issue for me since double click works. I'll leave it as your choice to leave the issue open or closed.
